docs/apv.md

```diff
@@ -352,9 +352,19 @@ De overtreding beschreven in lid 1 staat bekend als “spam”
 6. Het overtreden van lid 1,2,3 of 4. zal resulteren in een straf van de 1e categorie.
 7. Bij herhaalde overtredingen van lid 1,2,3 of 4 door een gang kan er worden overgegaan op gangstrikes.
 
+### Artikel 39 - Samenwerking
+
+1. Spelers die geen lid zijn van een gang mogen niet gebruikt worden om middelen ten behoeve van een gang te verzamelen
+of te verkopen zoals geld, drugs of wapens.
+2. Een gang mag maximaal vijf spelers met de rang loopjongen in hun gang hebben. Onder loopjongens worden officiële leden van de gang verstaan die zichzelf nog moeten bewijzen
+door middel van het verzamelen of verkopen van middelen ten behoeve van de gang.
+3. Een speler van een gang mag met bevriende spelers een roleplay doen zolang dit niet ten behoeve van zijn gang is.
```

Comment:
Voorbeeld toevoegen: Een voorbeeld hierin is dat je als speler prima met een vriend of kennis mag wit wassen voor jezelf, Echter zodra hier geld van je gang bij zit mag dit weer niet.